lumicks/pylake/kymotracker/kymotrack.py

```diff
@@ -764,9 +764,7 @@ def _validate_single_linetime_pixelsize(self):
         line_times_err_msg = (
             ""
             if len(line_times) == 1
-            else (
-                "All source kymographs must have the same line times, " f"got {line_times} seconds."
-            )
+            else ("All source kymographs must have the same line times, got {line_times} seconds.")
```

Comment:
Not an `f-string`. Would be nice to also add a test for this.

Reply:
good catch. I was playing fast and loose with the regex for the actual values (would accept anything between the `{}`) because `set` is not ordered. I sorted it now so that we can guarantee the exact message, but without some extra formatting, we end up with `got [0.5, 1] seconds` instead of `got {0.5, 1} seconds`. I guess here it's not necessary to stick to the strict `set` repr in the message if you're ok with that

Reply:
Right. Regarding the random order of `_kymos`, I wonder if that's not gonna be more of a headache than its worth (see later comment). I predict that this pattern will come up frequently and it may be wise to nip it in the bud now.